app/javascript/hooks/mutations/admin/site_settings/useUpdateSiteSetting.jsx

```diff
@@ -5,10 +5,23 @@ import axios from '../../../../helpers/Axios';
 export default function useUpdateSiteSetting(name) {
   const queryClient = useQueryClient();
 
+  // TODO - samuel: replace the create avatar form with this simple validation method
+  const imageValidation = (image) => {
+    const FILE_SIZE = 3_000_000;
+    const SUPPORTED_FORMATS = 'image/jpeg|image/png|image/svg';
+
+    if (image.size > FILE_SIZE) {
+      throw new Error('File is too large');
+    } else if (!image.type.match(SUPPORTED_FORMATS)) {
+      throw new Error('File type is not supported');
+    }
+  };
```

Comment:
This is needed as the FilesDragAndDrop component checks for the file type when _draggin' and droppin'_ the file, but not if the user clicks on the input and selects from it's files.

This should be added to the Presentation hook.

Also, this will replace the (overly complicated) create/update AvatarForm as we will get rid of the <Form> component, helpers, etc AND get rid of an extra button.

Reply:
This is indeed a good idea, we could extend it to be a helper that we can leverage throughout the app.
My only concern is how we're displaying all errors on the `onError` callback,

Reply:
I see what you mean.
Should we hardcode something like
`if not 'file too large' or 'file type is not supported' then send 'generic error message'`